pymeasure/instruments/srs/sr830.py

```diff
@@ -582,6 +638,30 @@ def get_buffer(self, channel=1, start=0, end=None):
         return self.binary_values("TRCB?%d,%d,%d" % (
             channel, start, end - start))
 
+    def get_buffer_bytes(self, channel=1, start=0, end=None):
+        """ Acquires the 32 bit floating point data through bytes transfer
+        This is the fastest transfer method according to the manual.
+        The byte to float conversion is provided in the manual
+        and implemented in :meth:`buffer_bytes_convert`
+        """
+        if end is None:
+            end = self.buffer_count
+        self.write("TRCL?%d,%d,%d" % (channel, start, end - start))
```

Comment:
In most cases it is better to use f-strings instead of this old percent style as it makes it easier to attribute the values to the position:
```suggestion
        self.write(f"TRCL?{channel},{start},{end - start}")
```